presto-docs/src/main/sphinx/release/release-0.193.rst

```diff
@@ -0,0 +1,76 @@
+=============
+Release 0.193
+=============
+
+General Changes
+---------------
+
+* Add :func:`map_zip_with` lambda function.
+* Add :func:`normal_cdf` function.
+* Add ``SET_DIGEST`` type and related functions.
+* Fix ``Tried to free more revocable memory than is reserved`` error for queries
+  that enable spilling and that are moved to the reserved memory pool.
+* Fix a bug that can cause the planner to go into an infinite loop. This bug was affecting queries
+  with multiple predicates that couldn't be pushed down to the connectors.
+* Fix a bug that may cause queries with ``row_number()`` to fail or return incorrect results.
```

Comment:
Fix a bug that causes queries that constrain the result of ``row_number()`` to numbers less than or equal to 0 to fail or return incorrect results.